includes/class-sensei-modules.php

```diff
@@ -2180,9 +2180,9 @@ public function setup_modules_taxonomy() {
 			'hierarchical'       => true,
 			'show_admin_column'  => false,
 			'capabilities'       => array(
```

Comment:
My first try was to just remove the `capabilities` to use the default ones. It would use just use the `manage_categories` for `edit_terms`.
But since teachers don't have the capability `manage_categories`, it was hiding the buttons to add the categories through the editor sidebar.